gneiss/plot/_dendrogram.py

```diff
@@ -49,7 +45,6 @@ class Dendrogram(TreeNode):
 
```

Comment:
This was a direct port from pycogent.  I'm very ok with dropping this.  Thanks for catching!